httpie/cli/definition.py

```diff
@@ -68,7 +70,8 @@
         $ http example.org hello=world   # => POST
 
     '''
-)
+).completer = ChoicesCompleter(
+    ('GET', 'HEAD', 'POST', 'PUT', 'PATCH', 'DELETE', 'OPTIONS'))
```

Comment:
Good evening. Is there a reason why it's not in the list: CONNECT, TRACE?

Reply:
CONNECT is proxy stuff only, not sure why one would use it directly with httpie (no idea whether it'd actually work). And TRACE is such a rarity that I don't think it deserves to be there. The thinking is to offer the most commonly used choices as completions. But if there's disagreement and CONNECT and TRACE actually work, sure they could be added.